memory-viz/src/automate.ts

```diff
@@ -73,7 +73,7 @@ function drawAutomatedStackFrames(stack_frames, configuration) {
         let width;
         let height;
 
-        if (stack_frame.name !== "BLANK") {
+        if (stack_frame.name !== ".blank") {
```

Comment:
Should now be `type` instead of `name` (unless I'm missing something?) A bit strange that the tests didn't catch this, perhaps that suggests new test(s) should be added?

Reply:
I believe this corresponds to drawing a blank stack frame (see the Blank Spaces example on the demo website).
The previous input would have been
`{ name: "BLANK", width: 100, height: 200, stack_frame: true }`
whereas the input is now
`{ type: ".frame", name: ".blank", width: 100, height: 200 }`.

I have confirmed that the output is the same by running the demo website on a previous commit. I have also added a note about blank stack frames in the documentation and added a snapshot test to check the behaviour!

Reply:
Ah interesting, great clarification. I don't like the mixing of `.blank` here though, so please modify the behaviour so that if the user wants to specify a blank stack frame, they use `type=".blank-frame"` instead (and don't pass in a `name`).